### PR TITLE
[10.0][MIG] account_move_locking

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

**Comment by sbidoul** [comment]
@grindtildeath @fclementic2c  Is this not superseded by account_permanent_lock_move? I kind of remember account_move_locking did only prevent the modification of existing moves but did not prevent the addition of moves in the past.

**Comment by fclementic2c** [comment]
@sbidoul yes you are right but account_move_locking is still necessary when you install account_cancel (to get the flexibility to do correction easily) but legally you must have a way to freeze the move at some point.

File: account_move_locking/README.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ In order to lock the journal entry, you need to follow this process:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/92/9.0
+   :target: https://runbot.odoo-community.org/runbot/92/10.0
```
Comment (jcdrubay) [inline account_move_locking/README.rst:22]: You also need to update the menus above:
Accounting > Adviser > Journal Entries

File: account_move_locking/__manifest__.py [hunk 2]
```diff
@@ -1,31 +1,12 @@
...
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-##############################################################################
```
Comment (elicoidal) [inline account_move_locking/__manifest__.py:19]: Please use short headers

File: account_move_locking/models/account.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline account_move_locking/models/account.py:1]: Please use short headers (everywhere)

**Review by elicoidal — CHANGES_REQUESTED** [review]
details

**Comment by grindtildeath** [comment]
@elicoidal @jcdrubay Fix according to your reviews.

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by yvaucher — APPROVED** [review]

**Pull request merged**